gno.land/pkg/gnoclient/client_txs.go

```diff
@@ -81,6 +98,78 @@ func (c *Client) Call(cfg CallCfg) (*ctypes.ResultBroadcastTxCommit, error) {
 	return c.signAndBroadcastTxCommit(tx, accountNumber, sequenceNumber)
 }
 
+// MultiCall executes a contract call on the blockchain.
+func (c *Client) MultiCall(cfg MultiCallCfg) (*ctypes.ResultBroadcastTxCommit, error) {
+	// Validate required client fields.
+	if err := c.validateSigner(); err != nil {
+		return nil, errors.Wrap(err, "validate signer")
+	}
+	if err := c.validateRPCClient(); err != nil {
+		return nil, errors.Wrap(err, "validate RPC client")
+	}
+
+	sequenceNumber := cfg.SequenceNumber
+	accountNumber := cfg.AccountNumber
+
+	msgs := make([]vm.MsgCall, 0, len(cfg.Msgs))
+	for _, msg := range cfg.Msgs {
+		pkgPath := msg.PkgPath
+		funcName := msg.FuncName
+		args := msg.Args
+		send := msg.Send
```

Comment:
Why the need to alias these vars?

Reply:
They are referenced more than once, but I removed the alias: [ebaf516](https://github.com/gnolang/gno/pull/1565/commits/ebaf516a36fc8c93705746c7feeea5126e0c439a)